packages/core/src/components/ColorPalette/VariantRow.tsx

```diff
@@ -134,16 +160,14 @@ export function VariantRow({
         l: val / 100,
       }).toHexString()
     }
-
-    const newToken = {
-      ...variant.token,
-      value: newColor,
-    }
-    onUpdateVariant({ ...variant, token: newToken }, isBase)
+    setColor(() => newColor)
```

Comment:
Nit: Why not just do 
```suggestion
    setColor(newColor)
```

?

Reply:
Yeah removed it. Its usually for lazy initialization. Was using it to test something. Thanks!